src/containers/blogs/Blog.scss

```diff
@@ -1,18 +1,20 @@
 /*Blog Cards Layout*/
+@use '../../globalColor' as v;
+
 .square {
   width: 440px;
   height: 430px;
-  background: white;
+  background: v.$textColorDark;
   border-radius: 4px;
-  box-shadow: 0px 20px 50px #d9dbdf;
+  box-shadow: 0px 20px 50px v.$darkBoxShadow;
   -webkit-transition: all 0.3s ease;
   -o-transition: all 0.3s ease;
   transition: all 0.3s ease;
   margin-top: 30px;
 }
 
 .square:hover {
-  box-shadow: 0 20px 40px rgba(0, 0, 0, 0.08);
+  box-shadow: 0 20px 40px v.$lightBoxShadow4;
```

Comment:
You should change this to `$lightBoxShadow2` and remove `$lightBoxShadow4` variable altogether as I mentioned in one the previous comments.